src/include/common/system_message.h

```diff
@@ -6,17 +6,22 @@
 
 namespace kuzu {
 namespace common {
+
 inline std::string systemErrMessage(int code) {
     // System errors are unexpected. For anything expected, we should catch it explicitly and
     // provide a better error message to the user.
     // LCOV_EXCL_START
     return std::system_category().message(code);
     // LCOV_EXCL_STOP
 }
+
 inline std::string posixErrMessage() {
     // LCOV_EXCL_START
     return systemErrMessage(errno);
     // LCOV_EXCL_STOP
 }
+
+std::string getDLError();
```

Comment:
let's remove `get` to be more consistent with other two functions. getDLError -> dlErrMessage().